rootfs/etc/profile.d/prompt.sh

```diff
@@ -168,9 +168,13 @@ function geodesic_prompt() {
 		KUBE_PS1_KUBECONFIG_CACHE=""
 	fi
 
+	local kube_prompt=""
+	if typeset -f "${kube_ps1}" > /dev/null; then
+		kube_prompt="$(kube_ps1)"
+	fi
+
 	if [ -n "${BANNER}" ]; then
-		# Intentional 2 spaces between banner mark and banner in order to offset it from level prompt
-		PS1=$' ${BANNER_MARK}'"  ${BANNER} $(kube_ps1)${secrets_active}\n${tf_prompt}${dir_prompt}"
+		PS1=$' ${BANNER_MARK}'"  ${BANNER} $(kube_prompt)${secrets_active}\n${tf_prompt}${dir_prompt}"
```

Comment:
You are wrapping `kube_prompt` with braces (command substitution)

```bash
 ✗ . [none] (HOST) geodesic ⨠
bash: kube_prompt: command not found
```

```suggestion
		PS1=$' ${BANNER_MARK}'"  ${BANNER} ${kube_prompt}${secrets_active}\n${tf_prompt}${dir_prompt}"
```